tests/kernels/test_cutlass.py

```diff
@@ -507,3 +508,137 @@ def test_cutlass_cuda_graph(per_act_token: bool, per_out_ch: bool):
 
 def test_cutlass_support_opcheck():
     opcheck(torch.ops._C.cutlass_scaled_mm_supports_fp8, (capability, ))
+
+
+# TODO add bias
+@pytest.mark.parametrize("num_experts", [8, 64])
+@pytest.mark.parametrize("per_act_token", [True, False])
+@pytest.mark.parametrize("per_out_ch", [True, False])
+@pytest.mark.parametrize("use_bias", [False])
+@pytest.mark.skipif(
+    (lambda x: x is None or not ops.cutlass_group_gemm_supported(x.to_int()))(
+        current_platform.get_device_capability()),
+    reason="Grouped gemm is not supported on this GPU type.")
+def test_cutlass_fp8_group_gemm(num_experts: int, per_act_token: bool,
+                                per_out_ch: bool, use_bias: bool):
```

Comment:
Could you address the TODO before we land this? I recently got bit by us not testing bias on the 2:4 sparse CUTLASS kernels so I'd like make sure we avoid that happening again...

Reply:
We don't need bias in fused_moe - I added this TODO when the gemm implementation was less MoE specific, so I think it's ok to remove it

Reply:
ok, sounds good, let's kill the TODO then :)